SourceCode/GPS/Forms/Pickers/FormColorPicker.Designer.cs

```diff
@@ -399,7 +399,7 @@ private void InitializeComponent()
         private System.Windows.Forms.Button btn00;
         private System.Windows.Forms.Button btn08;
         private System.Windows.Forms.CheckBox chkUse;
-        private System.Windows.Forms.GroupBox groupBox1;
+        private System.Windows.Forms.GroupBox labelGroupBoxSelectPresetColor;
```

Comment:
Shouldn't this just have the `groupBox` prefix?
```suggestion
        private System.Windows.Forms.GroupBox groupBoxSelectPresetColor;
```